pkg/operator/cluster/spec.go

```diff
@@ -39,3 +39,31 @@ type Spec struct {
 	// The path on the host where config and data can be persisted.
 	DataDirHostPath string `json:"dataDirHostPath"`
 }
+
+type PoolSpec struct {
```

Comment:
not sure on a long term pattern yet. maybe we even end up with a spec package. for now it's fine.